src/parade/LetterLink.stories.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import LetterLink from './LetterLink'
+
+storiesOf('LetterLink', module).add('default', () => (
+  <LetterLink letter="a" isActive />
```

Comment:
Can we make these props configurable from the UI? 😄

Reply:
ie both `letter` and `isActive` using [knobs](https://github.com/storybookjs/storybook/tree/master/addons/knobs)